src/modules/credentials/issuedList/issuedList.module.scss

```diff
@@ -13,6 +15,11 @@
         @apply px-6 py-3 align-middle text-sm font-normal text-Gray-light-mode-600 truncate;
         max-width: 180px;
     }
+
+    .row {
+        cursor: pointer;
+        border-bottom: 1px solid $color-primary-200;
```

Comment:
the double border at the end of the table for the last row (one related to `.row` and one related to `.paginationBox`)